### PR TITLE
Joss feedback @mbsuraj

### LINKED ISSUE #83: Community Guidelines Missing
This is part of [JOSS Review #10653](https://github.com/openjournals/joss-reviews/issues/10653) 

I notice that the package misses community guidelines for parties wishing to "1) Contribute 2) Report issues 3) Seek support".

Please provide those as a markdown file perhaps like `CONTRIBUTING.md`.

### LINKED ISSUE #84: Paper should clarify that benchmark variance is over model seeds only
This issue is for [JOSS Review #10653](https://github.com/openjournals/joss-reviews/issues/10653)

The paper states - "reported as mean ± standard deviation over 5 independent runs". However, when looking at benchmark.py, the train/test split is performed once i.e. `config_data(args.dataset, split_seed=args.seed)` before the run loop, and only the model initialization seed varies across the 5 runs. All runs share the same train / test partition. 

This means the reported ± std captures model variance (i.e. initialization and sampling stochasticity) but not data-split variance. This is a defensible experimental design - it isolates method stability from data-split noise - but the phrase "independent runs" could lead readers to assume full repeated random sub-sampling, which would typically produce wider confidence intervals. 

**Suggestion**: Clarify in the paper that the 5 runs use a fixed train/test split and vary only the model seed.

### LINKED ISSUE #85: BdeClassifier docstring incorrectly states default loss is GaussianNLL
This issue is for [JOSS Review #10653](https://github.com/openjournals/joss-reviews/issues/10653)

Inside bde.py, `BdeClassifier.__init__` docstring for the loss parameter incorrectly states GaussianNLL which is true for `BdeRegressor` but not for the classifier. 

The actual default for the classification (resolved in `FnnTrainer.default_loss`) is `CategoricalCrossEntropy`. 

**Suggested Fix**: Change the `BdeRegressor` docstring to 

```
loss : BaseLoss | None
    Custom training loss; defaults to :class:`bde.loss.CategoricalCrossEntropy`.

```

### LINKED ISSUE #86: ipython should not be a core dependency
This issue is for [JOSS Review #10653](https://github.com/openjournals/joss-reviews/issues/10653)

`pyproject.toml` lists `ipython>=8.0 ` under `[project] dependencies`, but it is not imported anywhere in the `bde/` source package.

This means every user who runs `pip install sklearn-contrib-bde` pulls IPython and its transitive dependencies for no functional benefit. 

**Suggestion**: Remove `"ipython>=8.0"` from the `dependencies` list in `pyproject.toml`. If it's needed for development or for testing, it already lives in the pixi lockfile anyway.

### LINKED ISSUE #87: Paper revisions needed: State of the Field, Software Design, and References
This issue is for [JOSS Review #10653](https://github.com/openjournals/joss-reviews/issues/10653)
  
For the JOSS sofware paper criteria, following paper sections need minor expansion:

1. **State of the Field**: name specific packages and add build-vs-contribute justificationThe current section refers to "several libraries" and "existing probabilistic programming frameworks" without naming them. JOSS requires that "when related tools exist, authors must provide a clear 'build vs. contribute' justification."

Please - 
a. Name the specific existing tools in this space (e.g., NumPyro, Pyro, TensorFlow Probability, Laplace-Redux, or similar BDL packages)
b. Explain briefly why a standalone package was built rather than contributing a MILE implementation to an existing framework like BlackJAX or NumPyro

2. **Software Design**: expand with architectural trade-offs
The current section is a single sentence about JAX parallelization. JOSS asks for "trade-offs weighed, the design/architecture chosen, and why it matters."

Please consider briefly addressing:

1. Why fully-connected networks only ? 
2. Why full-batch training ?
3. The two-stage design (optimization → sampling) ?

I believe two to three additional sentences would satisfy this requirement.

### PR DESCRIPTION
**Pull request by EmanuelSommer**
Address several issues raised by @mbsuraj. Namely: #83, #84, #85, #86, #87

### CODE REVIEW

**Review by mbsuraj — APPROVED** [review]
Appreciate the effort in addressing the issues for JOSS submission. These changes look good to me.
